### PR TITLE
Feature - Custom vision prediction and quick fixes

### PR DESCRIPTION
**Pull request by nrobert**
Implementation of **Custom Vision** predictions, for both **Classification** and **Object detection** projects: #6 

Impact:
- Move of `GetImageInfo` out of Face to be reusable between Analyzers

Fixes:
- Add select list's selected item to local storage (and default to first one if nothing in storage)

Various:
- SDK: 3.1.200 instead of 3.1.100

Notes for future features/bugs:
- API Exception management: if the wrong project type is selected for Custom Vision, the API will send an exception. This should be handled in the future, with a global exception management on all families of products (missing feature: API exceptions catch + error message display)
- Boxes bug when result boxes overlap (which must be similar in OCR and Face): we cannot access to details of the one which is below

### CODE REVIEW

File: src/Orneholm.CognitiveWorkbench.Web/Views/Vision/CustomVision.cshtml [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+                                            <div class="cwb-image-box" style="@obj.BoundingBox.ToCss()">
+                                                <div class="cwb-image-box-info" title="@obj.Probability.ToDescriptivePercentage()">
+                                                    <span class="cwb-image-box-type">Object</span>
+                                                    <span class="cwb-image-box-description">@obj.TagName</span>
+                                                </div>
+
+                                                <div class="cwb-image-box-metadata">
+                                                    <table class="cwb-image-box-metadata-table">
+                                                        <tbody>
+                                                            <tr class="cwb-image-box-metadata-table-header">
```
Comment (PeterOrneholm) [inline src/Orneholm.CognitiveWorkbench.Web/Views/Vision/CustomVision.cshtml]: I would remove the "Misc" header, as this is the only content we have.

**Review by PeterOrneholm — CHANGES_REQUESTED** [review]
Thanks! one small thing, see comment.

File: src/Orneholm.CognitiveWorkbench.Web/Views/Vision/CustomVision.cshtml [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+                                        The dimensions of the image must be greater than 256 pixels for both height and width (otherwise, they will be upscaled)
+                                    </li>
+                                    <li>
+                                        See service documentation for limits and quotas on <a href="https://docs.microsoft.com/en-us/azure/cognitive-services/Custom-Vision-Service/limits-and-quotas">docs.microsoft.com</a>
+                                    </li>
+                                </ul>
+                            </p>
+
+                            <p class="mb-3">
+                                <strong>Note:</strong> You need to provide your own keys for Azure Cognitive Services - Computer Vision under the tab <em>Settings</em>.
```
Comment (PeterOrneholm) [inline src/Orneholm.CognitiveWorkbench.Web/Views/Vision/CustomVision.cshtml]: COpy mistake here. Change to custom vision.

Comment (nrobert) [reply src/Orneholm.CognitiveWorkbench.Web/Views/Vision/CustomVision.cshtml]: You're right, a bad copy paste as the links were also wrong. Fixed

**Pull request merged**